Toivoa_Backend/Models/userModel.js

```diff
@@ -44,16 +44,11 @@ const userSchema = new Schema(
             type:String,
             required:true,
         },
-        passwordSalt:
-        {
-            type:String,
-            required:true,
-        },
-        accountType:
-        {
-            type:Number,
-            required:true,
-        },
+        accountType: {
+            type: String,
+            required: true,
+            enum: ['consumer', 'seller', 'market vendor'], // Define the choices for accountType
```

Comment:
Interesting change. We can keep it like this but you will have to update the mongodb database with a query then.